configure.py

```diff
@@ -1278,7 +1280,8 @@ def query_seastar_flags(seastar_pc_file, link_static_cxx=False):
                 f.write('build $builddir/{}/{}: ar.{} {}\n'.format(mode, binary, mode, str.join(' ', objs)))
             else:
                 objs.extend(['$builddir/' + mode + '/' + artifact for artifact in [
-                    'libdeflate/libdeflate.a'
+                    'libdeflate/libdeflate.a',
+                    'zstd/libzstd.a'
```

Comment:
Add a trailing comma so we don't have to touch the existing line next time.

Reply:
Done